modules/user_manual/pages/files/encrypting_files.adoc

```diff
@@ -91,7 +91,6 @@ Only the data in your files is encrypted, and not the filenames or
 folder structures. These files are never encrypted:
 
 * Old files in the trash bin.
-* Image thumbnails from the Gallery app.
```

Comment:
There are still thumbnails that are not encrypted. Suggest to remove "from the Gallery app".